assets/ui/spawnScreen.ui

```diff
@@ -8,100 +8,100 @@
         "contents" : [
             {
                 "type" : "UIBox",
-                "layoutInfo": {
+                "layoutInfo" : {
                     "width": 500,
                     "use-content-height": true,
                     "position-horizontal-center": {},
                     "position-vertical-center": {}
                 },
-                "content": {
-                    "type" : "ColumnLayout",
-                    "columns": 2,
-                    "verticalSpacing" : 8,
-                    "contents" : [
+                "content" :
```

Comment:
The { should remain on this line.

We use [1TBS](https://en.wikipedia.org/wiki/Indentation_style#Variant:_1TBS_(OTBS)). That means you you *always* have the `{` right next to the thing that you're starting - in this case, content. The `}` can, according to preference, go either in the same line, or in the next line. You've decided to go with next line, which is fine.